workspace/apps/pidp/src/app/features/portal/components/portal-card/portal-card.component.scss

```diff
@@ -2,7 +2,7 @@
 
 // Set parent variables
 .viewport-xsmall {
-  --gap: 1rem;
+  --gap: 2.3rem;
```

Comment:
is the change to the gap variable affecting any other elements of the layout? I noticed a change to the button padding
old
<img width="238" alt="gap in develop" src="https://github.com/bcgov/moh-pidp/assets/45132887/e3530749-2fa0-48d7-b2d6-669e7978e20a">
new
<img width="261" alt="new gap" src="https://github.com/bcgov/moh-pidp/assets/45132887/87413202-11f0-43dc-9be0-5d5ad26f9953">